examples/llava/clip.cpp

```diff
@@ -1489,10 +1530,15 @@ struct clip_ctx * clip_model_load(const char * fname, const int verbosity = 1) {
             LOG_INF("v_image_mean       %f %f %f\n", new_clip->image_mean[0], new_clip->image_mean[1], new_clip->image_mean[2]);
             LOG_INF("v_image_std        %f %f %f\n", new_clip->image_std[0], new_clip->image_std[1], new_clip->image_std[2]);
             LOG_INF("v_image_grid_pinpoints: ");
-            for (int i = 0; i < 32 && (hparams.image_grid_pinpoints[i] != 0); ++i) {
+            for (size_t i = 0; i < hparams.image_grid_pinpoints.size(); ++i) {
                 LOG_INF("%d ", hparams.image_grid_pinpoints[i]);
             }
```

Comment:
you can iterate using `for (const auto & elem : vector)`

```suggestion
            for (const auto & pp : hparams.image_grid_pinpoints) {
                LOG_INF("%d ", pp);
            }
```